biojava-structure/src/test/java/org/biojava/nbio/structure/secstruc/TestBasePairParameters.java

```diff
@@ -0,0 +1,31 @@
+import org.biojava.nbio.structure.secstruc.BasePairParameters;
+import org.junit.Test;
+
+/**
+ * Created by luke on 7/21/17.
+ */
+public class TestBasePairParameters {
```

Comment:
That had already been done with the commit "Made test use no println, and class use Logger"

I will rewrite it to use the Structure object alone (it would be the user's responsibility whether to use StructureIO or to use getStructure(filename) on a local file) and add the JavaDocs and other suggestions.  Thank you!  

--